base/strings/search.jl

```diff
@@ -253,7 +257,14 @@ julia> findnext("Lang", "JuliaLang", 2)
 6:9
 ```
 """
-findnext(t::AbstractString, s::AbstractString, i::Integer) = _search(s, t, i)
+function findnext(t::AbstractString, s::AbstractString, i::Integer)
+    if sizeof(t) == 1
```

Comment:
This cant be correct for `AbstractString`.

Reply:
It's definitely not. What is it trying to check for?

Reply:
Same as the comment above